po/cz.po

```diff
@@ -0,0 +1,163 @@
+# #-#-#-#-#  tukit.js.pot (Tukit 0.1)  #-#-#-#-#
+# tukit czech translation.
+# This file is distributed under the same license as the tukit package.
+# Zdenek Kubala <zkubala@suse.com>, 2022.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"#-#-#-#-#  tukit.html.pot (PACKAGE_VERSION)  #-#-#-#-#\n"
+"Project-Id-Version: tukit 0.1\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Cockpit html2po\n"
+"#-#-#-#-#  tukit.js.pot (PACKAGE_VERSION)  #-#-#-#-#\n"
```

Comment:
please simplify the metadata to remove duplicates. check `po/pl.po` for example

Reply:
I went through template file... maybe it could be simplified as well.

Reply:
The template is generated and updated automatically by the tools.